ow_lander/scripts/constants.py

```diff
@@ -11,19 +11,19 @@
 J_PROX_PITCH = 2
 J_SHOU_PITCH = 1
 J_SHOU_YAW = 0
+J_GRINDER = 5
 
 X_SHOU = 0.79
 Y_SHOU = 0.175
 HAND_Y_OFFSET = 0.0249979319838
 
 SCOOP_OFFSET = 0.215
-GRINDER_OFFSET = 0.8
+GRINDER_OFFSET = 0.16
 
 # Distance between scoop center of mass and lower blade
 SCOOP_HEIGHT = 0.076
-GRINDER_HEIGHT = 3*SCOOP_HEIGHT
 
-DEFAULT_GROUND_HEIGHT = -0.135
+DEFAULT_GROUND_HEIGHT = -0.155
```

Comment:
If this default applies to a particular terrain, or region of terrain, is the code safe?  See my earlier comment cautioning against specifying defaults in services.

Reply:
This is the terrain height found by guarded_move when running it with use_defaults=True in europa_terminator_workspace. I am not sure how safe it is to use it for other locations in europa_terminator, it might be OK since the terrain is not very irregular. It's definitely not safe to use it in atacama_y1a, since the terrain is very irregular there. However, autonomy shouldn't run the grinding service with use_defaults=True. Autonomy should know the ground height in that specific location and terrain, and use it to call the service.